README.md

```diff
@@ -190,7 +196,20 @@ Include one of these files on your page, and `ejs` should be available globally.
 Most of EJS will work as expected; however, there are a few things to note:
 
 1. Obviously, since you do not have access to the filesystem, `ejs.renderFile()` won't work.
-2. For the same reason, `include`s do not work.
+2. For the same reason, `include`s do not work unless you use an `IncludeCallback`. 
+Here is an example:
```

Comment:
@TimothyGu I added a newline because the line was already 85 chars long.

I will indent the code block though.